packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -163,5 +165,31 @@ describe('HttpValidator', () => {
         expect(headersValidator.validate).toHaveBeenCalled();
       });
     });
+
+    describe('cannot match status code with responses', () => {
+      it('returns error', () => {
+        (findResponseSpecModule.findOperationResponse as jest.Mock).mockReturnValue(undefined);
+        jest.spyOn(bodyValidator, 'validate').mockReturnValue([]);
+        jest.spyOn(headersValidator, 'validate').mockReturnValue([]);
+
+        expect(
+          validateOutput({
+            resource: {
+              method: 'get',
+              path: '/',
+              id: '1',
+              request: {},
+              responses: [{ code: '200' }],
+            },
+            element: { statusCode: 200 },
+          }),
+        ).toEqual([
+          {
+            message: 'Unable to match returned status code with those defined in spec',
```

Comment:
Why is this failing? Aren't we providing a 200 response which is indeed a listed response?

Reply:
Hah, we are mocking fetching of response here:
```
        (findResponseSpecModule.findOperationResponse as jest.Mock).mockReturnValue(undefined);
```

So thanks to beauty of unit tests it does not matter what's inside the spec. And sth needs to be there because of typing.

However, I agree that it may be misleading. I changed slightly the spec.